notch8/utk-hyku
# Browse Collections - All Collections Grid View

## Summary
Custom theming for Browse Collections section grid view only
- add collections list to controller
- add partials for browse collections section
- add initial styles
- update home content partial
- add browse collections section
- add collections to english config
- revert to hyrax default html file
- remove code for feature flipper elements
- move all code for featured collections into its partial
- move all code for browse collections into its partial
- add styles for browse collection grid section and update to use project variables
- add scroll div and styling
- update hyrax locales translations for featured collections and browse collections
- update admin theme notes regarding feature flippers for DC Repository theme

## Related Ticket
[Browse Collections - All Collections Grid View#61](https://github.com/scientist-softserv/utk-hyku/issues/61)

## Screencasts
### Full Screen

https://user-images.githubusercontent.com/39319859/207181182-b1dba980-105c-43da-9a4f-b38ccc79f0fd.mov

### Medium Screen

https://user-images.githubusercontent.com/39319859/207181233-0f110846-82dd-4a7d-a780-c6bf531891ac.mov

### Small Screen

https://user-images.githubusercontent.com/39319859/207181273-da59b0fa-9f2d-4848-94b2-56ca6c269965.mov

## QA Instructions
- These changes only apply to the DC Repository theme - all other themes should have default styling
- Test on staging at https://dc.utk-hyku-staging.notch8.cloud/
- NOTE: The Grid view button is hard coded as active until the toggle functionality is complete on next ticket.
- Test on browsers - specifically scroll of section
- Feature flippers should be _disabled_ for the following:
- [ ] Share work button
- Feature flippers should be _enabled_ for the following:
- [ ] Show featured works tab
- [ ] Show recently uploaded tab
- [ ] Show featured researcher
